ui/app/AppLayouts/Chat/panels/communities/ChatPermissionQualificationPanel.qml

```diff
@@ -0,0 +1,105 @@
+
+        StatusScrollView {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 20
+            contentWidth: tokenRow.implicitWidth
+            padding: 0
+            Row {
+                id: tokenRow
+                Repeater {
+                    id: repeater
+                    model: root.holdingsModel
+                    Row {
+                        spacing: d.panelRowSpacing
+                        Row {
+                            spacing: d.panelRowSpacing
+                            Repeater {
+                                model: HoldingsSelectionModel {
+                                    sourceModel: holdingsListModel
+                                    assetsModel: root.assetsModel
+                                    collectiblesModel: root.collectiblesModel
+                                }
+                                StatusListItemTag {
+                                    height: 20
+                                    enabled: false
+                                    leftPadding: 2
+                                    title: model.text
+                                    asset.name: model.imageSource
+                                    asset.isImage: true
+                                    asset.bgColor: "transparent"
+                                    asset.height: 16
+                                    asset.width: asset.height
+                                    asset.bgWidth: asset.height
+                                    asset.bgHeight: asset.height
+                                    asset.color: asset.isImage ? "transparent" : titleText.color
+                                    closeButtonVisible: false
+                                    titleText.color: model.available ? Theme.palette.primaryColor1 : Theme.palette.dangerColor1
+                                    bgColor: model.available ? Theme.palette.primaryColor2 :Theme.palette.dangerColor2
+                                    titleText.font.pixelSize: 12
+                                }
+                            }
+                        }
+                        StatusBaseText {
+                            height: parent.height
+                            visible: (index !== (repeater.count - 1))
+                            verticalAlignment: Text.AlignVCenter
+                            font.pixelSize: Style.current.primaryTextFontSize
+                            rightPadding: d.panelRowSpacing
+                            color: Style.current.secondaryText
+                            text: qsTr("or")
+                        }
+                    }
+                }
+            }
+        }
+
+        StatusBaseText {
+            Layout.preferredWidth: 30
+            Layout.fillHeight: true
+            font.pixelSize: Style.current.primaryTextFontSize
+            color: Style.current.secondaryText
+            verticalAlignment: Text.AlignVCenter
+            text: qsTr("or... ")
```

Comment:
I'm not sure what's the puprose of that "or..." label here. My presumption is to present some elision of the list and shouldn't be here statically all the time.

Now it will be here always, even where the condition is short.

Quick experiment with increasing the width:
![Screenshot from 2023-06-16 15-23-27](https://github.com/status-im/status-desktop/assets/20650004/898556bd-e2f8-4910-860f-0497660c1908)

Reply:
this or.. opens the popup (see video in description)

Reply:
Confirming this. The last `or..` should only be rendered when there's more token permissions that can not fit into the input anymore